packages/lib-classifier/src/store/DataVisAnnotatingStore.js

```diff
@@ -0,0 +1,37 @@
+import { types } from 'mobx-state-tree'
+
+const DataVisAnnotatingStore = types
+  .model('DataVisAnnotatingStore', {
+    active: types.optional(types.number, 0),
```

Comment:
Having a model for a single number feels heavy-handed. Is the general idea that each task will have a store to hold the state of the form for that task?

Reply:
I'm expecting some of the functions that are in the `LightCurveViewer` react component will move to this store when that is refactored to decouple the task from the viewer.

Reply:
Also, I think we do want to store the annotation form data in MST stores because then it validates against the model. We are doing this for the single and multiple choice task type annotations, too.